debian/changelog

```diff
@@ -1,6 +1,14 @@
+cuttlefish-common (0.9+nmu5) UNRELEASED; urgency=medium
+
+  * install_zip.sh: exit early in case of errors
+  * install_zip.sh: support partial extraction
```

Comment:
I'm not clear on where partial extraction and exiting early is added. The 'find "${destdir}" -name "*.img"' code only seems to do the sparse image conversion.

Reply:
That happened already here: https://github.com/google/android-cuttlefish/pull/6
This commit updates the change log including the missing entry from the mentioned PR.

Reply:
Sounds good, thanks.